packages/grafana-data/src/transformations/transformers/seriesToColumns.test.ts

```diff
@@ -45,58 +44,102 @@ describe('SeriesToColumns Transformer', () => {
       (received) => {
         const data = received[0];
         const filtered = data[0];
-        expect(filtered.fields).toEqual([
-          {
-            name: 'time',
-            state: {
-              displayName: 'time',
+        expect(filtered.fields).toMatchInlineSnapshot(`
+          Array [
+            Object {
+              "config": Object {},
+              "name": "time",
+              "state": Object {
+                "displayName": "time",
+              },
+              "type": "time",
+              "values": Array [
+                1000,
+                3000,
+                4000,
+                5000,
+                6000,
+                7000,
+              ],
             },
-            type: FieldType.time,
-            values: new ArrayVector([1000, 3000, 4000, 5000, 6000, 7000]),
-            config: {},
-            labels: undefined,
-          },
-          {
-            name: 'temperature',
-            state: {
-              displayName: 'temperature even',
+            Object {
+              "config": Object {},
+              "labels": Object {
+                "name": "even",
+              },
+              "name": "temperature",
+              "state": Object {
+                "displayName": "even temperature",
```

Comment:
`temperature even` vs `even temperature` is the real change here  --- BUT that does no actually make its way to the end user :shrug: